src/main.css

```diff
@@ -44,6 +44,7 @@
   .notifications.list-group a:hover, a.octolinker-link[href], .commit-id:hover,
   body[class="page-responsive"] .files-list .octicon,
   .timeline-comment-action:hover, .Header-link[data-ga-click]:focus,
+  .Header-link:hover, .Header-link:focus,
   .Header-link[data-ga-click]:hover, .btn-link .octicon-kebab-horizontal:hover {
```

Comment:
```suggestion
  .Header-link[data-ga-click]:hover, .btn-link .octicon-kebab-horizontal:hover,
  .Header-link:hover, .Header-link:focus {
```

Reply:
Now the biggest issue is which page is this at, blindly adding classes is not the right thing todo usually.